index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hamza Anwar</title>
+    <!-- css -->
+    <link rel="stylesheet" href="styles.css">
+    <!-- font aweosome -->
+    <script src="https://kit.fontawesome.com/d7195d9855.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <!-- wrapper -->
+    <div class="wrapper">
+
+        <!-- navbar -->
+        <nav class="navbar">
+            <div class="bars"><i class="fa-solid fa-bars fa-xl"></i></div>
+            <h6 class="title"><a href="#">Hamza Anwar</a></h6>
+            <div class="links">
+                <a href="#">Portofolio</a>
+                <a href="#">About</a>
+                <a href="#">Contact</a>
+                <a href="#"><i class="fa-solid fa-envelope fa-xl"></i></a>
+            </div>
+        </nav>
+        <!-- end of navbar -->
+
+        <!-- hero section -->
+        <section class="hero">
+
+            <!-- leftside -->
+            <div class="leftside">
+
+                <!-- social links -->
+                <div class="social-links">
+                    <span><a href="#"><i class="fa-brands fa-facebook-f fa-xl"></i></a></span>
+                    <span><a href="https://www.instagram.com/humzaanwar74/" target="blank"><i class="fa-brands fa-instagram fa-xl"></i></a></span>
+                    <span><a href="https://www.youtube.com/c/jsSolutions" target="blank"><i class="fa-brands fa-youtube fa-xl"></i></a></span>
+                    <span><a href="https://github.com/HamzaAnwar1998/" target="blank"><i class="fa-brands fa-github fa-xl"></i></a></span>
+                    <span><a href="https://www.linkedin.com/in/hamza-anwar-565563156/" target="blank"><i class="fa-brands fa-linkedin-in fa-xl"></i></a></span>
+                    <span><a href="https://twitter.com/anwarhamza919/" target="blank"><i class="fa-brands fa-twitter fa-xl"></i></a></span>
+                </div>
```

Comment:
[REQUIRED] Please kindly use the `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Please refer to the comment about [Sementics](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) to discover why.